packages/spor-react/src/theme/slot-recipes/dialog.ts

```diff
@@ -1,96 +1,229 @@
 import { defineSlotRecipe } from "@chakra-ui/react";
 import { bg } from "../utils/bg-utils";
 import { coreText } from "../utils/core-utils";
+import { dialogAnatomy } from "./anatomy";
 
 export const dialogSlotRecipe = defineSlotRecipe({
-  slots: [
-    "overlay",
-    "dialogContainer",
-    "dialog",
-    "header",
-    "closeButton",
-    "body",
-    "footer",
-  ],
-  className: "spor-modal",
+  slots: dialogAnatomy.keys(),
+  className: "spor-dialog",
   base: {
-    overlay: {
-      backgroundColor: "blackAlpha.600",
+    backdrop: {
+      bg: "blackAlpha.500",
+      pos: "fixed",
+      left: 0,
+      top: 0,
+      w: "100vw",
+      h: "100dvh",
       zIndex: "modal",
+      _open: {
+        animationName: "fade-in",
+        animationDuration: "slow",
+      },
+      _closed: {
+        animationName: "fade-out",
+        animationDuration: "moderate",
+      },
     },
-    dialogContainer: {
+    positioner: {
       display: "flex",
-      zIndex: "modal",
+      width: "100vw",
+      height: "100dvh",
+      position: "fixed",
+      left: 0,
+      top: 0,
+      "--dialog-z-index": "zIndex.modal",
+      zIndex: "calc(var(--dialog-z-index) + var(--layer-index, 0))",
```

Comment:
Du kan erstatte disse to med:
```suggestion
      zIndex: "modal",
```